core/services/mongodb-remote/src/main/java/com/mongodb/stitch/core/services/mongodb/remote/sync/internal/DataSynchronizer.java

```diff
@@ -231,12 +258,22 @@ public void start() {
           logger));
       if (syncThreadEnabled) {
         syncThread.start();
+        isRunning = true;
       }
     } finally {
       syncLock.unlock();
     }
   }
 
+  public void enableSyncThread() {
```

Comment:
[nit] Think this can be deleted.

Reply:
Having this here allows us to check that the thread is started from within our tests.